README.md

```diff
@@ -39,11 +39,11 @@ See [docker reference](https://rebase-helper.readthedocs.io/en/latest/user_guide
 |:------------------------- |:----------------------------------------------------------------- |
 | *report.txt*              | summary report with all important information                     |
 | *changes.patch*           | diff against original files, directly applicable to dist-git repo |
+| *logs*                    | logs with various verbosity levels                                |
```

Comment:
This should end with `/` to make it obvious that it's a directory.

Reply:
Also, the width of the first column could be adjusted to match the longest string, it is now unnecessary wide.